website/app/styles/doc-components/code-group.scss

```diff
@@ -0,0 +1,207 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+@use "../breakpoints" as breakpoint;
+@use "../typography/mixins" as *;
+
+:root {
+  --ember-shiki-border-radius: 0px;
+}
+
+.doc-code-group {
+  overflow: hidden;
+  border-radius: 4px;
+}
+
+.doc-code-group__action-bar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  border: 1px solid var(--doc-color-gray-400);
+  border-bottom: none;
+  border-top-left-radius: inherit;
+  border-top-right-radius: inherit;
+
+  .doc-code-group__language-picker {
+    display: flex;
+    gap: 8px;
+    align-items: center;
+    margin: 0;
+    padding: 6px;
+    border: none;
+    border-right: 1px solid var(--doc-color-gray-400);
+
+    .doc-code-group__language-picker-option {
```

Comment:
[note] usually we don't nest CSS selectors unless it's needed (lower specificity, easier to read, easier to refactor, etc)